homeassistant/components/dominos.py

```diff
@@ -0,0 +1,111 @@
+import json
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from pizzapi import Customer, Address, Order
+import unicodedata
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# The domain of your component. Should be equal to the name of your component.
+DOMAIN = 'dominos'
+
+ATTR_COUNTRY = 'country_code'
+ATTR_FIRST_NAME = 'first_name'
+ATTR_LAST_NAME = 'last_name'
+ATTR_EMAIL = 'email'
+ATTR_PHONE = 'phone'
+ATTR_ADDRESS = 'address'
+ATTR_ORDERS = 'orders'
+ATTR_DUMP_MENU = 'dump_menu'
+ATTR_ORDER_ENTITY = 'order_entity_id'
+
+REV = '183d0f5522e7fa99776e8c6c692b56e69f0e6b8f'
+REQUIREMENTS = [
+    'https://github.com/wardcraigj/pizzapi/archive/%s.zip#pizzapi==0.0.2'
+    % REV]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(ATTR_COUNTRY): cv.string,
+        vol.Required(ATTR_FIRST_NAME): cv.string,
+        vol.Required(ATTR_LAST_NAME): cv.string,
+        vol.Required(ATTR_EMAIL): cv.string,
+        vol.Required(ATTR_PHONE): cv.string,
+        vol.Required(ATTR_ADDRESS): cv.string,
+        vol.Optional(ATTR_ORDERS): cv.ensure_list,
+        vol.Optional(ATTR_DUMP_MENU): cv.boolean,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Set up is called when Home Assistant is loading our component."""
+    dominos = Dominos(hass, config)
+
+    hass.data[DOMAIN] = []
+
+    if config[DOMAIN].get(ATTR_DUMP_MENU):
```

Comment:
This should not be a config option.

Reply:
ok, no problem. Would it be preferable to perform this dump via a service call?